### DISCUSSION TITLE
Bringing models into a closed network

### DISCUSSION THREAD

**Discussion by mosescohen1** [openai/whisper#2001] [Q&A]
Hi everybody.

I am working on a closed, offline networ, and I was wondering if there is any way to all whisper models into my closed network, and have them work from there?

Thanks in advance

**Comment by glangford** [comment]
See these past discussions

* https://github.com/openai/whisper/discussions/1774
* https://github.com/openai/whisper/discussions/1463
* https://github.com/openai/whisper/discussions/1556
* https://github.com/openai/whisper/discussions/1399
